projects/1.20-fabric/assets/expanding-technologies/expandingtechnologies/lang/zh_cn.json

```
@@ -0,0 +1,32 @@
+{
+
+	"_": "->------------------------]  Game Elements  [------------------------<-",
+
+	"block.expandingtechnologies.railroad_light_controller": "铁路灯光控制器",
+
+
+	"_": "->------------------------]  UI & Messages  [------------------------<-",
+
+	"expandingtechnologies.display_source.railroad_light_controller_source_light_display": "灯光状态",
+
+
+	"_": "->------------------------]  Subtitles  [------------------------<-",
+
+	"subtitles.block.railroad_light_controller.crossing_bell": "铁路道口钟声",
+
+
+	"_": "->------------------------]  Ponder Content  [------------------------<-",
+
+	"expandingtechnologies.ponder.railroad_lights_instruction.header": "使用铁路灯光控制器",
+	"expandingtechnologies.ponder.railroad_lights_instruction.text_1": "使用铁路灯光控制器可以让铁路道口更美观",
+	"expandingtechnologies.ponder.railroad_lights_instruction.text_2": "要想使用控制器，首先连接一个以辉光管为目标的显示链接器",
+	"expandingtechnologies.ponder.railroad_lights_instruction.text_3": "然后，只需要用红石信号给控制器充能就可以打开灯光",
+	"expandingtechnologies.ponder.railroad_lights_instruction.text_4": "如果你在其上方放置一个音符盒，一段荷兰铁路钟声就会随着控制器被打开而响起",

```

这里的Dutch Railroad Bell有特殊指代吗
```suggestion
	"expandingtechnologies.ponder.railroad_lights_instruction.text_4": "在其上方放置音符盒，控制器打开时就会响起一段荷兰铁路钟声",
```